components/AnchorNavigation/src/index.scss

```diff
@@ -0,0 +1,46 @@
+.denhaag-anchor-navigation {
+  border-inline-start: var(--denhaag-anchor-navigation-border-inline-start);
+}
+
+.denhaag-anchor-navigation ul {
+  list-style: var(--denhaag-anchor-navigation-ul-list-style);
+  padding-inline-start: var(--denhaag-anchor-navigation-ul-padding-inline-start);
+}
+
+.denhaag-anchor-navigation ul li a {
+  font-family: var(--denhaag-anchor-navigation-anchor-font-family);
+  font-size: var(--denhaag-anchor-navigation-anchor-font-size);
+  line-height: var(--denhaag-anchor-navigation-anchor-line-height);
+  text-decoration: var(--denhaag-anchor-navigation-anchor-text-decoration);
+}
+
+.denhaag-anchor-navigation ul li a,
+.denhaag-anchor-navigation ul li a:visited {
+  color: var(--denhaag-anchor-navigation-anchor-color-default);
+}
+
+.denhaag-anchor-navigation ul li a:active,
+.denhaag-anchor-navigation ul li a:hover,
+.denhaag-anchor-navigation ul li a:target {
+  color: var(--denhaag-anchor-navigation-anchor-color-interactive);
+  position: relative;
+}
+
+.denhaag-anchor-navigation ul li a:active,
+.denhaag-anchor-navigation ul li a:target {
+  font-weight: var(--denhaag-anchor-navigation-anchor-font-weight);
+}
+
+.denhaag-anchor-navigation ul li a:active::before,
```

Comment:
Why not always add the `:before` and just change the background color on one of those CSS selectors.

Reply:
Because the grey border is set on the main component itself, the nav element for design reasons.
The border for the links are not property of the nav design.
These are only relevant when interacting with the links.